actionview/lib/action_view/railtie.rb

```diff
@@ -34,6 +34,8 @@ class Railtie < Rails::Railtie # :nodoc:
           ActionView::Resolver.caching = app.config.cache_classes
         end
       end
+
+      PartialRenderer.collection_cache = Rails.cache
```

Comment:
I'd expect this to use `config.action_controller.cache_store` (`ActionController::Base.cache_store`) since that's already used for fragment caching, and falls back to `Rails.cache` if unconfigured.